Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1575,12 +1575,25 @@ void TextureCacheBase::CopyRenderTargetToTexture(u32 dstAddr, EFBCopyFormat dstF
     return;
   }
 
-  const unsigned int tex_w = scaleByHalf ? srcRect.GetWidth() / 2 : srcRect.GetWidth();
-  const unsigned int tex_h = scaleByHalf ? srcRect.GetHeight() / 2 : srcRect.GetHeight();
+  u32 tex_w = width;
```

Comment:
Not sure if this is the right place for this (or if it should go into BPStructs) but can you add a comment for future readers on why there's a subtle difference between the 'normal' texture width/height and the scaled texture width/height?  (one uses a passed in value and the other uses the srcRect)

Reply:
Done